Lib/pathlib.py

```diff
@@ -1363,8 +1363,13 @@ def link_to(self, target):
 
     def rename(self, target):
         """
-        Rename this path to the given path,
-        and return a new Path instance pointing to the given path.
+        Rename / move this path to the target path.
+
+        The target path may be either absolute or relative. If given a relative
+        path, it'll be relative to the current working directory, *not* the
```

Comment:
```suggestion
        path, it'll be relative to the current working directory, not the
```
Docstrings don't have markup.